app/controllers/pxe_controller/iso_datastores.rb

```diff
@@ -114,7 +114,7 @@ def iso_datastore_get_form_vars
   # Set form variables for edit
   def iso_datastore_set_form_vars
     @edit = {}
-    @edit[:emses] = ManageIQ::Providers::Redhat::InfraManager.without_iso_datastores.order(:name)
+    @edit[:emses] = ManageIQ::Providers::Ovirt::InfraManager.without_iso_datastores.order(:name)
```

Comment:
We can do this in a follow-up but this should be switched to use supports so `ExtManagementSystem.supports?(:create_iso_datastore)` or something like that

Reply:
Sure, I'll take note for a future PR